### PR TITLE
Zendesk Support Form

### PR DESCRIPTION
**Pull request by grrowe**
- added /support route
- create form for user to submit a ticket

https://github.com/aptible/app-ui/assets/54414856/8394ca03-839b-44b7-8d06-b3a8827b8692

### CODE REVIEW

File: src/ui/shared/application-sidebar.tsx [hunk 1]
```diff
@@ -153,7 +153,7 @@ export const ApplicationSidebar = () => {
             </a>
             <a
```
Comment (neurosnap) [inline src/ui/shared/application-sidebar.tsx:154]: `<Link to={supportUrl()}>...</Link>`

**Review by neurosnap — COMMENTED** [review]
This looks pretty close to me.  Is `<SupportPage />` only allowed for authenticated users?

**Comment by grrowe** [comment]
> This looks pretty close to me. Is `<SupportPage />` only allowed for authenticated users?

Initially yes. There are future plans to make it anonymous but for now, yes.

**Review by joeclayallday — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -58,7 +58,8 @@
     "redux": "^4.2.1",
     "redux-batched-actions": "^0.5.0",
     "redux-persist": "^6.0.0",
-    "saga-query": "^8.0.0",
+    "rome": "^12.1.1",
```
Comment (neurosnap) [inline package.json:61]: remove this dependency

File: src/ui/shared/application-sidebar.tsx [hunk 3]
```diff
@@ -154,12 +155,18 @@ export const ApplicationSidebar = () => {
             >
               DOCS
             </a>
-            <a
+            {/* <a
```
Comment (neurosnap) [inline src/ui/shared/application-sidebar.tsx:158]: remove

File: yarn.lock
Comment (neurosnap) [inline yarn.lock:1]: make sure this doesn't include anything from `rome`

File: package.json (hunk 2 above)
Comment (grrowe) [reply package.json]: removed

File: yarn.lock
Comment (grrowe) [reply yarn.lock:1]: cleaned up yarn.lock

File: .yarn/releases/yarn-3.6.3.cjs
Comment (neurosnap) [inline .yarn/releases/yarn-3.6.3.cjs:1]: Why do we need this?  You didn't install any dependencies so I'm not sure we need this.

File: .yarnrc.yml [hunk 4]
```diff
@@ -3,3 +3,5 @@ nodeLinker: node-modules
 plugins:
   - path: .yarn/plugins/@yarnpkg/plugin-interactive-tools.cjs
     spec: "@yarnpkg/plugin-interactive-tools"
+
+yarnPath: .yarn/releases/yarn-3.6.3.cjs
```
Comment (neurosnap) [inline .yarnrc.yml:7]: why do we need this?

File: yarn.lock
Comment (neurosnap) [inline yarn.lock:1]: We should have fewer changes here.

File: .yarnrc.yml (hunk 4 above)
Comment (grrowe) [reply .yarnrc.yml:7]: yarn auto generated this when I ran `yarn [set version](https://yarnpkg.com/cli/set/version) latest`

File: .yarn/releases/yarn-3.6.3.cjs
Comment (neurosnap) [reply .yarn/releases/yarn-3.6.3.cjs:1]: https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored

File: .yarnrc.yml (hunk 4 above)
Comment (neurosnap) [reply .yarnrc.yml:7]: https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored

File: yarn.lock
Comment (neurosnap) [reply yarn.lock:1]: https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored

**Review by neurosnap — APPROVED** [review]

**Pull request merged**